stable/prometheus/templates/server-statefulset.yaml

```diff
@@ -187,3 +182,28 @@ spec:
           configMap:
             name: {{ .configMap }}
       {{- end }}
+{{- if and .Values.server.persistentVolume.enabled (not .Values.server.persistentVolume.existingClaim) }}
+  volumeClaimTemplates:
```

Comment:
@unguiculus Thanks for the review again, and yes, you are right!

In fact, I tested and the `volumeClaimTemplates` object doesn't even seem to support the `claimName` field (which `existingClaim` uses). So it doesn't seem possible to reuse the PVCs this way. However, that shouldn't be a concern/problem since the behavior of `StatefulSets` is reuse existing PVCs anyway.

I've made the appropiate changes and removed the `existingClaim` field from the values, templates, and docs.